src/media_tools/webvtt.c

```diff
@@ -430,7 +443,6 @@ GF_Err wvtt_dump(GF_Box *a, FILE * trace)
 	GF_WebVTTSampleEntryBox *cuebox = (GF_WebVTTSampleEntryBox *)a;
 	gf_isom_box_dump_start(a, "WebVTTSampleEntryBox", trace);
 	fprintf(trace, ">\n");
-	if (cuebox->config) boxstring_dump((GF_Box *)cuebox->config, trace);
```

Comment:
Yes. I believe `gf_isom_box_dump_done()` prints it by default. I.e.: I saw MP4Box -diso report something like:
```
                <WebVTTConfigurationBox Size="19" Type="vttC" Specification="p30" Container="wvtt"><![CDATA[
WEBVTT FILE
]]></WebVTTConfigurationBox>
                <WebVTTConfigurationBox Size="19" Type="vttC" Specification="p30" Container="wvtt"><![CDATA[
WEBVTT FILE
]]></WebVTTConfigurationBox>
              </WebVTTSampleEntryBox>

```
even though there's obviously only one vttC. I wasn't sure if removal was the appropriate solution, but it results in a single vttC being written